src/_community/rules.md

```diff
@@ -46,14 +46,38 @@ Essentially, please take requests from community team members seriously and list
 something. If you have a problem with what a team member is asking you to do, we advise that you step away from the 
 situation and send a private message to another team member.
 
-#### 5) Keep all projects legal & appropriate
-
-If you need help with a project, you will not be provided with it if your project may break laws, breach a Terms of
-Service or EULA (belonging to any organisation, not just the platforms we're using), or be considered malicious or 
-inappropriate.
-
-Additionally, keep your code appropriate. Do not sexualize your code or use any slurs or other forms of derogatory
-language, either in the code itself or in the assets or filenames that make up your project.
+#### 5) Keep all projects legal, legitimate & appropriate
+
+While anyone may make use of the Quilt toolchain in accordance with the licenses applied to each project, all projects 
+developed or promoted in Quilt community spaces must meet the following criteria:
+
+* Projects must not be malicious - which includes (but is not limited to) malware, mods that include backdoors, or 
+  mods that harass specific people or groups of people (for example, by automatically kicking them from servers with 
+  the mod installed)
+* Projects must not be sexualised, hateful or otherwise inappropriate in nature, and this also applies to all code 
+  and project assets posted for any purpose
+* Projects must not break laws, including copyright laws - this includes piracy tools, as well as projects that 
+  ignore or otherwise break the licenses of other projects
+* Projects must not break the Terms of Service or EULA of any organisation, including (but not limited to) Mojang, 
+  Microsoft, Discord and any other community platforms we use
+* Projects must not hide their functionality, and should be upfront with what they do - however, projects with 
```

Comment:
What for is this? Not unreasonable, but seems completely useless. If you don't document your projects, it's your problem only - as well as the consequences of that

Reply:
It's intent I believe is to avoid mods that would do something without the user's knowledge, eg. An ores mod regenerating chunks that it detects you only branch mined through or something

Reply:
I think what is meant here is that mods don't have undocumented - potentially malicious - functionalities which a user cannot possible know without digging through the code

Reply:
This intent is suitable for guidelines, not rules

Reply:
That statement doesn't make sense to me - why would this not be suitable for a ruleset?

As others have pointed out, the point of this part of the rule is that mods shouldn't deliberately behave contrary to what they claim to be - in a sense, that in itself would be malicious. I'm not sure why that sentiment isn't something that should be enshrined here.

Reply:
Hiding might be subjective. What you describe is deceiving and lying - not hiding. To alter the meaning, you may just remove a comma `Projects must not hide their functionality and should be upfront with what they do`. I agree this may be a nitpick, but when it comes to legislating - any nitpicks are relevant

Reply:
I disagree that using `hiding` here doesn't cover malicious scenarios, but removing the comma is acceptable.